doc/examples/applications/plot_3d_interaction.py

```diff
@@ -59,7 +59,7 @@
 print(f'range: ({vmin}, {vmax})')
 
 #####################################################################
-# We turn to `plotly`'s implementation of the `imshow` function, for it
+# We turn to ``plotly``'s implementation of the `imshow` function, for it
```

Comment:
```suggestion
# We turn to Plotly's implementation of the :func:`plotly.express.imshow` function, for it
```
Add Plotly to our intersphinx mapping? Also I don't see why Plotly should be a code literal here.